.release.cloudbuild.yaml

```diff
@@ -243,14 +243,14 @@ steps:
     docker push gcr.io/ml-pipeline/google/pipelines-test/visualizationserver:$(cat /workspace/mm.ver)
 
 - name: 'gcr.io/cloud-builders/docker'
-  args: ['pull', 'gcr.io/tfx-oss-public/ml_metadata_store_server:0.14.0']
+  args: ['pull', 'gcr.io/tfx-oss-public/ml_metadata_store_server:v0.21.0']
```

Comment:
qq: I know their version is `v0.21.0` which is a bit of inconsistent with typical versioning convention.

Do you want me to change it to 0.21.0?

Reply:
I think it would be better to match the released mlmd tags. I do agree its `v0.21.0` is odd and felt the same. Unless, Marketplace has requirement, I would prefer to keep it that way.

Reply:
Retagged to 'gcr.io/tfx-oss-public/ml_metadata_store_server:0.21.0'. Feel free to update in this PR.

Reply:
Yes you're right. Actually that does not make any difference b/c MKP only sees versions from our repo.